README.md

```diff
@@ -15,8 +15,8 @@
 
 ## What is Rook?
 
-Rook is a distributed storage system designed for cloud native applications. It
-exposes file, block, and object storage on top of shared resource pools. Rook has minimal
+Rook is a distributed storage system designed for cloud native applications. Rook interoperates with Kubernetes, to provide block, file system and object storage to its applications.
+Rook exposes file, block, and object storage on top of shared resource pools. Rook has minimal
 dependencies and can be deployed in dedicated storage clusters or converged clusters. It's
 self-managing, self-protecting, self-healing, and is designed to just work without teams of
 engineers managing it. It scales from a single node, to multi-PB clusters spread geographically.
```

Comment:
We may want to add a message around why K8s developers should care about Rook.  Something like this:

> Rook aims to be the storage solution for container-native app developers. Just like your container apps, Rook is designed to take advantage of the resource management of orchestration platform, Kubernetes. This allows Rook to share cluster resources with your apps, allowing it to dynamically scale as your apps scale and save the effort to manage storage infrastructure separately.  Rook provides persistent storage for your Kubernetes apps - which is one of the largest barriers for developers to fully containerize their apps.